blazor/predefined-dialogs/position.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Positioning in Blazor Predefined Dialog Component | Syncfusion
+description: Checkout and learn here all about positioning in Syncfusion Predefined Dialogs component and much more details.
+platform: Blazor
+control: Predefined Dialogs
+documentation: ug
+---
+
+# Positioning in Blazor Predefined Dialogs Component
+
+Customize the Dialog position by using the [DialogOptions.Position](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogOptions.html#Syncfusion_Blazor_Popups_DialogOptions_Position) property. The position can be represented with specific `X` and `Y` values.
+
+* The `PositionDataModel.X` can be configured with a left, center, right, or offset value. By default, the value is set as `center`.
+* The `PositionDataModel.Y` can be configured with a top, center, bottom, or offset value. By default, the value is set as `center`.
```

Comment:
* The `PositionDataModel.Y` can be configured with a top, center, bottom, or offset value. By default, the value is set as `center.`